src/LibraryManager/LibraryNaming/VersionedLibraryNamingScheme.cs

```diff
@@ -26,15 +28,16 @@ class VersionedLibraryNamingScheme : ILibraryNamingScheme
                 return (name, version);
             }
```

Comment:
add this to simplify handing a single-character libraryId (simplifies below):
```
else if(libraryId.Length == 1)
{
    return (libraryId, version);
}
```

Reply:
Fixed in commit 2